package.json

```diff
@@ -25,21 +25,22 @@
     "cuid": "^2.1.6",
     "debug": "^4.1.0",
     "graphql-extensions": "^0.5.2",
+    "graphql-parse-resolve-info": "^4.4.5",
     "graphql-tag": "^2.10.1",
     "graphql-tag-pluck": "^0.8.7",
     "graphql-toolkit": "^0.5.0",
     "graphql-tools": "^4.0.5",
     "lodash.set": "^4.3.2"
   },
   "peerDependencies": {
-    "graphql": "^14.0"
+    "graphql": "^14"
```

Comment:
this was ultimately my fault (from when I added in the peerDependency on graphql), but I think to be a compete valid semver string this should be "^14.0.0" (same goes for the dev dependency below). It may not matter but yeah figured I'd call it out.